frontend/src/components/App/NotificationIcon.js

```diff
@@ -1,6 +1,11 @@
 import React, { PropTypes, Component } from 'react'
 
 class NotificationIcon extends Component {
+
+  static propTypes = {
+    unreadNotificationsCount: PropTypes.number
+  }
+
```

Comment:
filename comment:

This is under App, but maybe it should be called AppNavbar or something? Implying it's part of the elements at the top/sides/bottom of the app